src/wp-admin/includes/upgrade.php

```diff
@@ -2293,6 +2297,28 @@ function upgrade_600() {
 	}
 }
 
+
+/**
+ * Executes changes made in WordPress 6.3.0.
+ *
+ * @ignore
+ * @since 6.3.0
+ *
+ * @global int $wp_current_db_version The old (current) database version.
+ */
+function upgrade_630() {
+	global $wp_current_db_version;
+
+	if ( $wp_current_db_version < 55752 ) {
+		if ( ! is_multisite() ) {
+			// Replace non-autoload option can_compress_scripts with autoload option, see #55270
+			$can_compress_scripts = get_option( 'can_compress_scripts' ) ? 1 : 0;
+			delete_option( 'can_compress_scripts' );
+			add_option( 'can_compress_scripts', $can_compress_scripts );
```

Comment:
```suggestion
			update_option( 'can_compress_scripts', $can_compress_scripts, 'yes' );
```
Q: Instead of deleting and adding new option can we use `update_option`?